source/Applications/Advanced/HandEyeCalibration/README.md

```diff
@@ -13,26 +13,27 @@ To fully understand Hand-Eye Calibration, please see the [tutorial](https://zivi
    4. Repeat 1.-3. until 10-20 pose pairs are collected
    5. Enter command to perform calibration and return a **Transformation Matrix**
 
-[ZividHandEyeCalibration](C:\Program Files\Zivid\bin\ZividHandEyeCalibration.exe): (no source)
+[ZividHandEyeCalibration](https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/96469274)
```

Comment:
why not use the same standard here as elsewhere? I mean use ZividHandEyeCalibration-url here and then at the bottom of the file add the address.

Reply:
Done, standardized all links now